### PR TITLE
coreographer to choreographer

### PR DESCRIPTION
**Pull request by yhedholm**

### CODE REVIEW

File: backend/src/main/java/com/google/rolecall/Constants.java [hunk 1]
```diff
@@ -32,11 +32,11 @@
 
   public static class Roles {
     public static final String ADMIN = "ADMIN";
-    public static final String COREOGRAPHER = "COREOGRAPHER";
```
Comment (mikzat) [inline backend/src/main/java/com/google/rolecall/Constants.java:35]: Is there any impact on the DB from this change?

File: frontend/rolecall/src/app/user/user-editor.component.ts [hunk 2]
```diff
@@ -72,7 +72,7 @@ export class UserEditor implements OnInit {
   rolesNamesMap = {
     isAdmin: 'Is Admin',
     // TODO: Spelling error in `choreographer`.
```
Comment (mikzat) [inline frontend/rolecall/src/app/user/user-editor.component.ts:74]: I believe the TODO will be removed from the other PR, but wouldn't hurt to remove it here as well.

**Review by mikzat — APPROVED** [review]

File: backend/src/main/java/com/google/rolecall/Constants.java (hunk 1 above)
Comment (yhedholm) [reply backend/src/main/java/com/google/rolecall/Constants.java:35]: No. It seems as if the original developers changed strategy at some point and ignored their system wide constants. CHOREOGRAPHER only shows up in one place, inside a comment!

File: frontend/rolecall/src/app/user/user-editor.component.ts (hunk 2 above)
Comment (yhedholm) [reply frontend/rolecall/src/app/user/user-editor.component.ts:74]: Done.

**Pull request merged**